scripts/client/lint-rust.mjs

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env zx
 import 'zx/globals';
-import { workingDirectory } from '../utils.mjs';
+import { getClippyToolchain, workingDirectory } from '../utils.mjs';
 
-// Check the client using Clippy.
+// Check the client using nightly clippy.
 cd(path.join(workingDirectory, 'clients', 'rust'));
-await $`cargo clippy ${process.argv.slice(3)}`;
+await $`cargo +${getClippyToolchain()} clippy ${process.argv.slice(3)}`;
```

Comment:
Should this check that there was a toolchain specified otherwise default to `stable`?

Reply:
Oh good point. Even better, we can have this default to the empty string so it picks up the system default is rather than `stable` -- what do you think?

Reply:
Perfect!